lake/pool.go

```diff
@@ -327,7 +327,7 @@ func (p *Pool) ScanSegments(ctx context.Context, w zio.Writer, snap *commit.Snap
 		close(ch)
 	}()
 	m := zson.NewZNGMarshaler()
-	m.Decorate(zson.StyleSimple)
+	m.Decorate(zson.StylePackage)
```

Comment:
Is this important?

Reply:
There's a collision between index.Reference and segment.Reference- needs to include package scope to differentiate between the two.